lib/ui/addPet/AddPetProfileScreen.dart

```diff
@@ -0,0 +1,395 @@
+import 'dart:io';
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:myportion_app/model/Feeder.dart';
+import 'package:myportion_app/model/Pet.dart';
+import 'package:myportion_app/model/Schedule.dart';
+import 'package:myportion_app/services/FirestoreUtils.dart';
+import 'package:myportion_app/ui/home/HomeScreen.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:myportion_app/services/helper.dart';
+import 'package:intl/intl.dart'; //for date format
+
+import '../../constants.dart' as Constants;
+import '../../constants.dart';
+import '../../main.dart';
```

Comment:
## Paths

These should be changed to absolute paths

```dart
import 'package:myportion_app/constants.dart' as Constants;
import  'package:myportion_app/main.dart';
```

## Necessity

Do we need to import Constants twice?